public/app/features/alerting/components/AlertingQueryEditor.tsx

```diff
@@ -104,9 +118,38 @@ export class AlertingQueryEditor extends PureComponent<Props, State> {
     );
   }
 
+  isRunning() {
+    const data = Object.values(this.state.panelDataByRefId).find((d) => !!d);
```

Comment:
nit: Maybe use Boolean instead of !!
```suggestion
    const data = Object.values(this.state.panelDataByRefId).find((d) => Boolean(d));
```